src/plugins/integrations/AWSIntegration.ts

```diff
@@ -330,6 +338,7 @@ class AWSIntegration implements Integration {
 
         } catch (error) {
           ThundraLogger.getInstance().error(error);
+          return wrappedFunction.apply(this, [callback]);
```

Comment:
Can we be sure that in case of error `wrappedFunction.apply` is not called twice

Reply:
In case of error in the wrapped function error callback is called. I verified, it is not called twice.